corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html

```diff
@@ -9,7 +9,6 @@
         </label>
         <div class="controls" style="padding-top: 5px;">
             <input type="hidden"
-                   name="{{ field.html_name }}"
```

Comment:
@orangejenny Is this related to https://github.com/dimagi/commcare-hq/pull/9395 ?